@DrQuackeroo  Oops, CI failed.

We can't remove an explicit constructor due to  DEFAULT_METHOD macro https://github.com/syoyo/tinygltf/blob/a5e653e46cca64bbc0ecee7ede007347a94c39b0/tiny_gltf.h#L75 , sorry 😅

```
./tiny_gltf.h:6208:16: error: no matching constructor for initialization of 'Accessor'
      Accessor accessor;
```

Could you please revert the change https://github.com/syoyo/tinygltf/pull/516/commits/1b517f2b23036e23f8c28d35ab2594ae8398e296 ?

--- a/tiny_gltf.h
+++ b/tiny_gltf.h
@@ -898,6 +898,7 @@ struct Accessor {
     // unreachable return 0;
   }
 
+  Accessor() = default;
   DEFAULT_METHODS(Accessor)
   bool operator==(const tinygltf::Accessor &) const;
 };

